crucible-syntax/src/Lang/Crucible/Syntax/ExprParse.hs

```diff
@@ -34,8 +34,9 @@ import Control.Monad.IO.Class (MonadIO)
 import Control.Monad.Reader (MonadReader(..), ReaderT(..))
 import qualified Control.Monad.State.Strict as Strict
 
+import Data.Containers.ListUtils (nubOrd)
```

Comment:
Since we're now using this, make sure to bump the lower version bounds on `containers` to `>= 0.6.0.1` in the `.cabal` file.